apps/src/templates/sectionAssessments/StudentMultipleChoiceOverviewTable.story.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import StudentMultipleChoiceOverviewTable from './StudentMultipleChoiceOverviewTable';
+import {studentOverviewData} from './assessmentsTestHelpers';
+
+export default storybook => {
+  return storybook
+    .storiesOf('SectionAssessments/StudentMultipleChoiceOverviewTable', module)
+    .addStoryTable([
+      {
+        name: 'Student overview',
+        description: 'Display assessment summary for each student',
```

Comment:
Someone who doesn't know about the assessments tab spec should be able to understand what this table is for, so I think this description needs to be more detailed.

Reply:
Fixed.  Thanks